dnf/module/module_base.py

```diff
@@ -104,11 +104,14 @@ def install(self, module_specs, strict=True):
         for pkg_name, set_specs in install_dict.items():
             query = install_base_query.filter(name=pkg_name)
             if not query:
-                for spec in set_specs:
-                    logger.error(_("Unable to resolve argument {}").format(spec))
-                logger.error(_("No match for package {}").format(pkg_name))
-                error_specs.extend(set_specs)
-                continue
+                # package could be available from non-modular repository
```

Comment:
I'd probably change the text to something like following:
# package can also be non-modular or part of another stream